pyomo/contrib/cspline_external/cspline_parameters.py

```diff
@@ -11,6 +11,8 @@
 
 from pyomo.common.dependencies import numpy as np
 import pyomo.environ as pyo
+from pyomo.core.base.block import BlockData, declare_custom_block
+import idaes
```

Comment:
We do not allow `idaes` to be imported in Pyomo because `idaes` depends on Pyomo. There are lots of potentially bad circular import issues that can occur.

Reply:
Oops, put that there to set the solver path.  Meant to delete it.